### PR TITLE
Suggestion Update faq.md

### PR DESCRIPTION
**Pull request by manashar4**
@JV0812 - Sharing this suggestion as a generic reference for any administrators interested in understanding how the system optimizes the query. 

Could you please improve the experience of the same and add it to the help documentation.

## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/manage/partitions/flex/faq.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ The table below should give you a sense of how the number of partitions you use
 | 1/2 day          | 2                            | 2 TB         |
 | 2 days           | 2                            | 8 TB         |
 
+< add this as a note > 
+Given all partition definitions and the user query, SUMO performs automatic query rewriting optimizations by rewriting the query by including _view/_index clauses and tries to use the minimal list of views the system has to go through to serve the query. This list of views is what is eventually considered as part of scanned volume (actual or estimated).
```
Comment (kimsauce) [inline docs/manage/partitions/flex/faq.md:80]:
```suggestion
:::note 
Given all partition definitions and the user's query, Sumo Logic automatically optimizes query execution by rewriting the query to include `_view` and `_index` clauses. The system then identifies the minimal list of views required to serve the query, which ultimately contributes to the scanned volume, whether actual or estimated.
:::
```

Comment (kimsauce) [reply docs/manage/partitions/flex/faq.md:80]: @manashar4 lmk what you think

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor tweaks. @manashar4, thanks for submitting!

**Review by jpipkin1 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@kimsauce, your comments have been addressed. Could you approve?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**